jetty-client/src/test/java/org/eclipse/jetty/client/HttpClientAuthenticationTest.java

```diff
@@ -676,7 +675,7 @@ public void testTestHeaderInfoParsing() {
         Assert.assertTrue(headerInfos.get(0).getType().equalsIgnoreCase("Newauth"));
         Assert.assertTrue(headerInfos.get(0).getParameter("realm").equals("apps"));
         Assert.assertTrue(headerInfos.get(0).getParameter("type").equals("1"));
-        Assert.assertThat(headerInfos.get(0).getParameter("title"), Matchers.equalTo("Login to \"apps\""));
+        Assert.assertThat(headerInfos.get(0).getParameter("title"), Matchers.equalTo("Login to \\\"apps\\\""));
```

Comment:
@gregw the behaviour of this test differs to what was accepted previously, not sure which version is correct

Reply:
The \\\" version looks wrong to me.

Reply:
That's the version with slosh slosh slosh quote

Reply:
this is technically illegal by the rfc anyway, but all the quote stuff is handled within QuotedCSV

Reply:
I'm not sure that QuotedCSV will expand the quoted values if you have retain quotes on.    I think this might be an issue due to the quoted characters not being handled (see above).